packages/common/array/src/has-items.ts

```diff
@@ -1,7 +1,8 @@
 import { Nullable } from '@angular-ru/common/typings';
 
 import { hasNoItems } from './has-no-items';
+import { ListType } from './types';
 
-export function hasItems<EntryType>(array?: Nullable<EntryType[]>): array is [EntryType, ...EntryType[]] {
-    return !hasNoItems(array);
+export function hasItems<EntryType>(list?: Nullable<ListType<EntryType>>): boolean {
```

Comment:
I think is not good changes, because we have `array` secondary package, but now this api supported multiple containers

also changed `array is [EntryType, ...EntryType[]]` to `boolean`  that means is breaking changes

@Schirbak what do you think?

@mansurishmuhametov for more readable api, you need create `@angular-ru/common/collections/keyed`

```ts
import { hasKeyedItems } from '@angular-ru/common/collections';
import { hasManyKeyedItems } from '@angular-ru/common/collections';
// ...
```

where `hasKeyed*` support  only `Map`, `Set`, `WeakMap`, `WeakSet`

Reply:
maybe that was not the best idea
lets postpone it for a while